pkg/sources/github/github.go

```diff
@@ -879,6 +881,10 @@ func (s *Source) addReposForMembers(ctx context.Context, reporter sources.UnitRe
 // addUserGistsToCache collects all the gist urls for a given user,
 // and adds them to the filteredRepoCache.
 func (s *Source) addUserGistsToCache(ctx context.Context, user string, reporter sources.UnitReporter) error {
+	if s.ignoreGists {
+		return nil
+	}
```

Comment:
Man I was gonna say "let's do this check at the call site" but there's like 5-6? So, nevermind then haha.

Reply:
Yeah 😄 I thought of that at first as well, but it’s used in quite a few places.